tools/tasks/seed/check.tools.ts

```diff
@@ -0,0 +1,30 @@
+import * as util from 'gulp-util';
+import * as runSequence from 'run-sequence';
+
+import { join } from 'path';
+
+let fs = require('fs');
```

Comment:
that's what i had originally but one of the CI builds failed saying the constants didn't exist there. it did work fine locally tho, and it passed the other CI builds. so maybe there is an issue in a particular node version or something weird?